tunneling/nextjs/pages/api/tunnel.js

```diff
@@ -0,0 +1,25 @@
+import { withSentry } from "@sentry/nextjs";
+import axios from "axios";
+
+async function handler(req, res) {
+  const envelope = req.body;
+  const pieces = envelope.split("\n");
+
+  if (!pieces[0]) {
+    return res.status(200).json({ status: "broken_envelope" });
```

Comment:
With 200 one might not spot the body indicating an issue. What about something in the 4xx range?

```suggestion
    return res.status(400).json({ status: "broken_envelope" });
```

Reply:
Thanks @bruno-garcia, good point! I was thinking between 400 and 422 and forgot to choose between them 😄